cat-launcher/src/pages/PlayPage/ReleaseNotesDialog.tsx

```diff
@@ -0,0 +1,46 @@
+import { useQuery } from "@tanstack/react-query";
+
+import { ConfirmationDialog } from "@/components/ui/ConfirmationDialog";
+import { GameVariant } from "@/generated-types/GameVariant";
+import { fetchReleaseNotes } from "@/lib/commands";
+import { queryKeys } from "@/lib/queryKeys";
+
+interface ReleaseNotesDialogProps {
+  variant: GameVariant;
+  releaseTagName: string;
+  open: boolean;
+  onOpenChange: (open: boolean) => void;
+  onConfirm: () => void;
+  confirmText: string;
+}
+
+export function ReleaseNotesDialog({
+  variant,
+  releaseTagName,
+  open,
+  onOpenChange,
+  onConfirm,
+  confirmText,
+}: ReleaseNotesDialogProps) {
+  const { data: releaseNotes, isLoading } = useQuery({
```

Comment:
In cat-launcher/src/pages/PlayPage/ReleaseNotesDialog.tsx around lines 25 to 29,
the useQuery call does not handle query errors so a failed fetch is
indistinguishable from "no release notes". Add error handling by destructuring
error and isError from useQuery, add an onError callback to log/report the
error, and update the component UI to render a distinct error message (and
optionally a retry button that calls refetch) when isError is true; keep
existing loading and empty-state behavior only for isLoading and when
releaseNotes is empty.